test/cli/allow-target-env/push-changed-config-values/args

```diff
@@ -0,0 +1 @@
+push "my change description" --force --storage-api-token %%TEST_KBC_STORAGE_API_TOKEN%%
```

Comment:
Do we have to use `--force`? If yes change name of  test to `push-force-changed-config-values` and create another test without `--force`.